swift/ios-shoppe-demo/Views/LoginCollectionViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginCollectionViewController.swift
+//  ios-shoppe-demo
+//
+//  Created on 8/18/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+
+import UIKit
+
+class LoginCollectionViewController: UITableViewController {
+
+    var userName: String?
+    var userEmail: String?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        registerCellsNibs()
+    }
+
+    // Register cell classes
+
+    func registerCellsNibs() {
+        navigationItem.title = "Login"
+        navigationController?.navigationBar.tintColor = .white
+        registerCells(tableView: tableView, registry: loginViewControllerRegistry)
+    }
+
+    func login() {
+        guard let userName = userName,
```

Comment:
nit but perhaps it makes more sense to move the `let userEmail` to the same line so it's clearer the guard applies to both.  Was looking at this in [SO](https://stackoverflow.com/questions/39198626/multiple-conditions-in-guard-statement-swift)

```
guard let var2 = str["asd"], let var1 = str["status"]
    else
    {
        print("asdsfddffgdfgdfga")
        return
    }
```

Reply:
Having `else` on its own line is unconventional. I do agree that having them be on the same line would look nicer, so perhaps it should be:

```swift
guard let userName = userName, let userEmail = userEmail else { 
    return 
}
```